generators/Generators.fs

```diff
@@ -197,6 +197,25 @@ type PerfectNumbers() =
         | "deficient" -> "(Some Deficient)"
         | _ -> "None"
 
+type PascalsTriangle() =
+    inherit Exercise()
+
+    override this.RenderSutProperty canonicalDataCase = "triangle"
```

Comment:
Could you remove this to just use the default property name?

Reply:
Changed to default property name "rows" and parameter name to "numberOfRows"